content/en/docs/instrumentation/php/_index.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: PHP
-description: A language-specific implementation of OpenTelemetry in PHP.
+description: >-
+  <img width="35" src="https://github.com/open-telemetry/opentelemetry.io/blob/main/iconography/32x32/PHP.svg"></img>
```

Comment:
Update with URL to raw.github:

https://raw.githubusercontent.com/open-telemetry/opentelemetry.io/main/iconography/32x32/PHP.svg

Reply:
Done